### PR TITLE
ctpdev: adding input counters to CCDB

### PR DESCRIPTION
**Pull request by lietava**
Replacing Detector Counters in CTP scalers by Input counters needed for lumi (ZNC).

### CODE REVIEW

File: DataFormats/Detectors/CTP/src/RunManager.cxx [hunk 1]
```diff
@@ -134,6 +135,14 @@ int CTPRunManager::addScalers(uint32_t irun, std::time_t time)
       // LOG(info) << "Scaler for detector:" << countername << ":" << detcount;
     }
   }
+  */
+  int NINPS = 48;
+  int offset = 599;
+  for (uint32_t i = 0; i < NINPS; i++) {
+    uint32_t inpcount = mCounters[offset + i];
+    scalrec.scalersInps.push_back(inpcount);
```
Comment (martenole) [inline DataFormats/Detectors/CTP/src/RunManager.cxx:143]: @shahor02 but here @lietava is adding some counters to the scalers, no?

Comment (shahor02) [reply DataFormats/Detectors/CTP/src/RunManager.cxx:143]: Ops, sorry, missed that part, perhaps was checking only particular commits. But then the ClassDef of CTPScalerRecordRaw and CTPScalerRecordO2 need to be incremented.

**Comment by davidrohr** [comment]
@lietava : Is this needed at P2?
What is it for?

File: DataFormats/Detectors/CTP/src/RunManager.cxx (hunk 1 above)
Comment (lietava) [reply DataFormats/Detectors/CTP/src/RunManager.cxx:143]: Data members were not changed. I reused existing data member

Comment (davidrohr) [reply DataFormats/Detectors/CTP/src/RunManager.cxx:143]: But data members are changed... `scalersDets` is removed and `scalersInps` is added!

**Comment by shahor02** [comment]
Checks were already passed, merging after incrementing ClassDefs.

**Comment by davidrohr** [comment]
ok, should be squash-merged to get rid of the formatting and fixup commits

**Comment by shahor02** [comment]
Ah, sh., sorry, merged with rebase only, do you want me to revert it and merge with squashing, so that a single commit can be cherry-picked?

**Comment by martenole** [comment]
Its ok as is, we can cherry-pick ranges. But in general its of course better to squash unless commits are separated logically from one another

**Pull request merged**